Integrations/Exabeam-new/Exabeam-new.yml

```diff
@@ -0,0 +1,219 @@
+    - default: false
+      description: limit number of returned results.
+      isArray: false
+      name: limit
+      required: true
+      secret: false
+    deprecated: false
+    description: Get notable users in a period of time.
+    execution: false
+    name: get-notable-users
+    outputs:
+    - contextPath: Exabeam.NotableUser.RiskScore
+      description: Notable user risk score.
+      type: Number
+    - contextPath: Exabeam.NotableUser.UserFullName
+      description: The user's full name.
+      type: String
+    - contextPath: Exabeam.NotableUser.AverageRiskScore
+      description: Notable user avarage risk score.
+      type: Number
+    - contextPath: Exabeam.NotableUser.Executive
+      description: Is the user executive
+      type: Boolean
+    - contextPath: Exabeam.NotableUser.FirstSeen
+      description: Date of the user first seen
+      type: String
+    - contextPath: Exabeam.NotableUser.NotableSessionIds
+      description: Notable session ID.
+      type: String
+    - contextPath: Exabeam.NotableUser.AccountsNumber
+      description: Number of accounts.
+      type: Number
+    - contextPath: Exabeam.NotableUser.LastSeen
+      description: When the user last seen.
+      type: Unknown
+    - contextPath: Exabeam.NotableUser.Location
+      description: User's location.
+      type: String
+    - contextPath: Exabeam.NotableUser.UserName
+      description: User name.
+      type: String
+    - contextPath: Exabeam.NotableUser.Labels
+      description: User's labels.
+      type: String
+    - contextPath: Exabeam.NotableUser.LastActivityType
+      description: User last actibity type.
+      type: String
+  - deprecated: false
+    description: Get all watchlist ids and titles.
+    execution: false
+    name: get-watchlist
+    outputs:
+    - contextPath: Exabeam.Watchlist.Category
+      description: The watchlist category.
+      type: String
+    - contextPath: Exabeam.Watchlist.Title
+      description: The watchlist title.
+      type: String
+    - contextPath: Exabeam.Watchlist.WatchlistID
+      description: Watchlist ID.
+      type: String
+  - deprecated: false
+    description: Get all peer groups.
+    execution: false
+    name: get-peer-groups
+    outputs:
+    - contextPath: Exabeam.PeerGroup.Name
+      description: The name of the peer group.
+      type: String
+  - arguments:
+    - default: false
+      description: The title of the watchlist
+      isArray: false
+      name: title
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: The watchlist category.
+      isArray: false
+      name: category
+      predefined:
+      - PeerGroups
+      - UserLabels
+      required: true
+      secret: false
+    - default: false
+      description: Watchlist descripiton.
+      isArray: false
+      name: description
+      required: false
+      secret: false
+    - default: false
+      description: Items to add to the watchlist.
```

Comment:
it is not clear what items can be added, please add an example and mention that CSV is supported